libs/langgraph/langgraph/prebuilt/tool_node.py

```diff
@@ -43,11 +45,15 @@
 TOOL_CALL_ERROR_TEMPLATE = "Error: {error}\n Please fix your mistakes."
 
 
+class InvalidToolCommandError(Exception):
```

Comment:
Are there other exceptions in pre-built?

Should it really be sub-classing from Exception?

What does it mean for a Command returned by a tool to be invalid?

Reply:
figured out how to avoid this altogether!